operator/bundle_helpers/requirements.in

```diff
@@ -1,2 +1,2 @@
 pyyaml==6.0.2
-pytest==7.0.1
```

Comment:
@mclasmeier `pytest` can be removed from this file because the one used in GHA and local builds has `pytest` (where it probably needs to be bumped).
https://github.com/stackrox/stackrox/blob/master/operator/bundle_helpers/requirements-gha.txt